haddock-api/src/Haddock/Utils.hs

```diff
@@ -326,16 +326,10 @@ makeAnchorId (f:r) = escape isAlpha f ++ concatMap (escape isLegal) r
 
 
 jsFile :: String
```

Comment:
Could you rename jsFile to something more meaningful? Now that we have several JavaScript includes this derserves a less generic name I think. How about haddockJsFile?

Reply:
Fixed.